src/Randomizer.Data/WorldData/Regions/Zelda/LightWorld/LightWorldNorthWest.cs

```diff
@@ -51,7 +52,8 @@ public LightWorldNorthWest(World world, Config config, IMetadataService? metadat
                 memoryAddress: 0xE2,
                 memoryFlag: 0x9,
                 metadata: metadata,
-                trackerState: trackerState);
+                trackerState: trackerState)
+                .Weighted(100);
```

Comment:
I don't want to start another rework, but a LocationBuilder would be nice, and a good alternative to the many overloads we have now...